service/matching/config.go

```diff
@@ -149,7 +152,10 @@ type (
 		// Retry policy for fetching user data from root partition. Should retry forever.
 		GetUserDataRetryPolicy backoff.RetryPolicy
 
-		loadCause loadCause
+		loadCause                   loadCause
+		BreakdownMetricsByTaskQueue func() bool
+		BreakdownMetricsByPartition func() bool
+		BreakdownMetricsByBuildID   func() bool
```

Comment:
maybe keep the "settings" separate from loadCause?
```suggestion
		BreakdownMetricsByTaskQueue func() bool
		BreakdownMetricsByPartition func() bool
		BreakdownMetricsByBuildID   func() bool

		loadCause loadCause
```